packages/smart-wallet/src/smartWallet.js

```diff
@@ -725,17 +971,18 @@ export const prepareSmartWallet = (baggage, shared) => {
           return this.state.updateRecorderKit.subscriber;
         },
         getPublicTopics() {
-          const { currentRecorderKit, updateRecorderKit } = this.state;
+          const { state } = this;
+
           return harden({
             current: {
               description: 'Current state of wallet',
-              subscriber: currentRecorderKit.subscriber,
-              storagePath: currentRecorderKit.recorder.getStoragePath(),
+              subscriber: state.currentRecorderKit.subscriber,
+              storagePath: state.currentRecorderKit.recorder.getStoragePath(),
             },
             updates: {
               description: 'Changes to wallet',
-              subscriber: updateRecorderKit.subscriber,
-              storagePath: updateRecorderKit.recorder.getStoragePath(),
+              subscriber: state.updateRecorderKit.subscriber,
+              storagePath: state.updateRecorderKit.recorder.getStoragePath(),
```

Comment:
unnecessary change. please revert

Reply:
Agree they have to come out of state. I think this will read more clearly but not a blocker
```js
          const { state } = this;
          const { currentRecorderKit, updateRecorderKIt } = state;
```